packages/backend/src/api/routers/StatusRouter.ts

```diff
@@ -1,21 +1,37 @@
 import Router from '@koa/router'
+import { UnixTime } from '@l2beat/common'
+import { ParsedUrlQuery } from 'querystring'
 
 import { StatusController } from '../controllers/status/StatusController'
 
 export function createStatusRouter(statusController: StatusController) {
   const router = new Router()
 
   router.get('/status/prices', async (ctx) => {
-    ctx.body = await statusController.getPricesStatus()
+    const { from, to } = getFromTo(ctx.query)
+
+    ctx.body = await statusController.getPricesStatus(from, to)
   })
 
   router.get('/status/balances', async (ctx) => {
-    ctx.body = await statusController.getBalancesStatus()
+    const { from, to } = getFromTo(ctx.query)
+
+    ctx.body = await statusController.getBalancesStatus(from, to)
   })
 
   router.get('/status/reports', async (ctx) => {
-    ctx.body = await statusController.getReportsStatus()
+    const { from, to } = getFromTo(ctx.query)
+
+    ctx.body = await statusController.getReportsStatus(from, to)
   })
 
   return router
 }
+
+function getFromTo(query: ParsedUrlQuery): { from: UnixTime; to: UnixTime } {
```

Comment:
I would have from and to be possibly undefined and then StatusController could set this to Clock.getLastHour() and Clock.getFirstHour()